fdbclient/StorageServerInterface.h

```diff
@@ -310,6 +318,43 @@ struct GetKeyValuesRequest : TimedRequest {
 	}
 };
 
+struct GetKeyValuesAndFlatMapReply : public LoadBalancedReply {
```

Comment:
I think it worth noting that if one changes GetKeyValuesAndFlatMapRequest/Reply, it has to change GetKeyValuesRequest/Reply as well.

Reply:
Good point. Will add a comment in a different PR. And I really wish I can find out a way to make templates work better with Flow.